src/cheri/insns/gchi_32bit.adoc

```diff
@@ -21,6 +21,9 @@ include::wavedrom/gchi.adoc[]
 Description::
 Copy the metadata (bits [CLEN-1:MXLEN]) of capability `cs1` into `rd`.
 
-Operation::
+Prerequisites::
+{cheri_base_ext_name}
```

Comment:
I removed the Prequisites part since it's in the RV32CH/RV64CH chapter so clearly defined by that extension, but if you think we need it can add it back.